pyproject.toml

```diff
@@ -101,14 +101,11 @@ pyvista = [
     'uniform.vtk',
 ]
 
-[tool.isort]
-profile = 'black'
-line_length = 100
```

Comment:
Wee need `line_length` configuration and it is discussed in https://github.com/charliermarsh/ruff/issues/3206 .